docs/2.9.0/docs/canton/usermanual/FAQ.rst

```diff
@@ -163,6 +163,19 @@ system:
 
 .. literalinclude:: /canton/includes/mirrored/community/app/src/test/resources/documentation-snippets/dev-version-support.conf
 
+.. _how-do-i-enable-preview-features:
+
+How do I enable preview features?
```

Comment:
Can we change the customer visible term from "preview" to "Beta" or "Initial Access"?  I don't want to add another term unless absolutely necessary.  I do believe that "Beta" or "Initial Access" is the right term based on the [2024-4 Product Delivery Guidelines](https://docs.google.com/document/d/16nJHy_Efpd4ivsSsYcWpS4UeddjUUOVneuId-3AjiiI/edit#heading=h.tw9km21kaef4).